src/modules/AddressBook.js

```diff
@@ -0,0 +1,238 @@
+   *  };
+   * 
+   *  function addressBookSaved(err, responce) {
+   *    if(err) {
+   *      
+   *    } else {
+   *      console.log('All constacts saved');
+   *    }
+   *  }
+   * 
+   *  QB.addressbook.cud(addressBookList, savedAddressBookCallback);
+   *  // or second parameters can be options
+   *  QB.addressbook.cud(addressBookList, options, savedAddressBookCallback);
+   * 
+   */
+  cud: function(list, optionsOrcallback, callback) {
+    if (!Array.isArray(list)) {
+      new Error('First parameter must be an Array.');
+      return;
+    }
+
+    var opts, cb;
+
+    if(isFunction(optionsOrcallback)) {
+      cb = optionsOrcallback;
+    } else {
+      opts = optionsOrcallback;
+      cb = callback;
+    }
+
+    var data = { contacts: list };
+
+    if(opts) {
+      if(opts.force) {
+        data.force = opts.force;
+      }
+
+      if(opts.udid) {
+        data.udid = opts.udid;
+      }
+    }
+
+    this.service.ajax({
+      'type': 'POST',
+      'dataType': 'json',
+      'url': Utils.getUrl(config.urls.addressbook),
+      'data': data,
+      'contentType': 'application/json; charset=utf-8',
+      'isNeedStringify': true
+    },function(err, res) {
+      if (err) {
+        cb(err, null);
+      } else {
+        cb(null, res);
+      }
+    });
+  },
+
+  _isFakeErrorEmptyAddressBook: function(err) {
+    var errDetails = err.detail ? err.detail : err.message.errors;
+    return err.code === 404 && errDetails[0] === 'Empty address book';
+  },
+
+  /**
+   * This callback is called `getContactsCallback` and passed 2 arguments: err and responce.
+   * @callback getContactsCallback
+   * @param {Object} err 
+   * @param {Object[]} responce
+   */
+
+  /**
+   * Retrive all contacts from address book.
+   * The methods accepts 1 or 2 parameters.
+   * @memberof QB.addressbook
+   * @param {string|function} udidOrCallback - You could pass udid of address book or
+   * callback function if you want to get contacts from global address book.
+   * @param {getContactsCallback} [callback] - Callback function uses as 2nd parameter if you pass udid as 1st parameters.
+   * This callback 2 arguments: error and responce.
+   *
+   * @example
+   * var UDID = 'D337E8A4-80AD-8ABA-9F5D-579EFF6BACAB';
+   * 
+   * function gotContacts(err, contacts) {
+   *  contacts.forEach( (contact) => { alert(contact); })
+   * }
+   * 
+   * QB.addressbook.get(gotContacts);
+   * // or you could specify what address book you need by udid 
+   * QB.addressbook.get(UDID, gotContacts);
+   */
+  get: function(udidOrCallback, callback) {
+    var self = this;
+    var udid, cb;
+
+    if(isFunction(udidOrCallback)) {
+      cb = udidOrCallback;
+    } else {
+      udid = udidOrCallback;
+      cb = callback;
+    }
+
+    if(!isFunction(cb)) {
+      throw new Error('The QB.addressbook.get accept callback function is required.');
+    }
+
+    var ajaxParams = {
+      'type': 'GET',
+      'dataType': 'json',
```

Comment:
Is it need to set 'json'? [dataType has 'json' by default](https://github.com/QuickBlox/quickblox-javascript-sdk/blob/cad1e05874ef0a95138011042ddd20dae52242b0/src/qbProxy.js#L82)